Pamukkale Üniversitesi Fen Bilimleri Enstitüsü (Türkçe)

rsnbhr opened the pull request:
This citation style is a revision of the "American Psychological Association 6th edition (Türkçe)" style 
(http://www.zotero.org/styles/apa-tr), adapted for Pamukkale University The Graduate School of Natural and 
Applied Sciences. 

Two main modifications were applied:
1. In-text citations with three or more authors always use "et al." from the very first citation. 
2. In the bibliography, all authors are always listed (no use of "et al.").

rsnbhr commented on pau-fbe-tr.csl:
```diff
@@ -0,0 +1,504 @@
+    <id>http://www.zotero.org/styles/pau-fbe-tr</id>
+    <link href="http://www.zotero.org/styles/pau-fbe-tr" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa-tr" rel="template"/>
+    <link href="https://github.com/citation-style-language/styles/issues/240" rel="documentation"/>
```
Thank you for the feedback.
I am quite new to CSL development, so please excuse me if my question sounds a bit basic. May I ask if the <link rel="documentation"> field is mandatory? If it is not required, I can simply remove it.

This style is a revision of the "apa-tr" style (https://github.com/citation-style-language/styles/blob/master/apa-tr.csl), with two modifications tailored for use at Pamukkale University The Graduate School of Natural and Applied Sciences (Turkey):

1. In-text citations: for works with three or more authors, always use “et al.” from the first citation.

2. Bibliography: always list all authors (no “et al.”).

I am happy to update it with a more suitable link if you recommend one.

rsnbhr replied:
I had thought about it a bit differently. My initial plan was to prepare this citation style first and then publish it on our Graduate School of Natural and Applied Sciences website. Now I understand what you meant — we should first publish the official rules on our website and then submit the style from there. Thank you for clarifying this. I’ll prepare it shortly and update the link accordingly.

POBrien333 replied:
Great. Thanks for clarifying. Let us know once that is ready.

POBrien333 replied:
So, I don't fully understand why you would host the .csl file on your on servers and then want to have it on this repository?
The beauty of this repository is that you can have one single link pointing to the style on zotero.org/styles once it's published and users can also find it from within the softwares that use the CSL repository. Being an open-source project it also allows that other members can fix issues and the style so receives future updates.
Having it in two places is certainly not the ideal way to go.

POBrien333 commented on pamukkale-universitesi-fen-bilimleri-enstitusu.csl:
```diff
@@ -0,0 +1,474 @@
+                    <date variable="accessed">
+                      <date-part name="day" suffix=" "/>
+                      <date-part name="month" suffix=" "/>
+                      <date-part name="year" suffix=" tarihinde "/>
```
What does this tarihinde do? Can this be replaced with a term/label?

rsnbhr replied:
Regarding "tarihinde": It roughly translates to "on the date of" in Turkish. It provides the necessary grammatical connection (suffix) for the date. I believe keeping it this way ensures the sentence flows correctly in Turkish. As for the other changes, I reverted some of them to keep the output consistent with the format required by the university. Thanks for your help.

POBrien333 commented:
That's not how this works. Those changes I made were deliberate. Please revert them. 
If you spot an issue, please communicate that, don't just revert our work that is made with many years of expertise.

rsnbhr commented on pamukkale-universitesi-fen-bilimleri-enstitusu.csl:
```diff
@@ -20,38 +20,15 @@
-      <term name="and">ve</term>
-      <term name="in">içinde</term>
-      <term name="retrieved">erişildi</term>
+      <term name="et-al">v.d.</term>
```
"<term name="et-al">v.d.</term>" should be "<term name="et-al">vd.</term>"

rsnbhr commented on pamukkale-universitesi-fen-bilimleri-enstitusu.csl:
```diff
@@ -20,38 +20,15 @@
-      <term name="in">içinde</term>
-      <term name="retrieved">erişildi</term>
+      <term name="et-al">v.d.</term>
+      <term name="retrieved">erişildi veri tabanından</term>
```
"<term name="retrieved">erişildi veri tabanından</term>" should be "<term name="retrieved">erişildi</term>"

rsnbhr commented on pamukkale-universitesi-fen-bilimleri-enstitusu.csl:
```diff
@@ -175,11 +153,13 @@
-          <text variable="number" prefix=" No: "/>
-        </group>
+          <group delimiter=": ">
+            <label variable="number" form="short" text-case="capitalize-first"/>
```
<label variable="number" form="short" text-case="capitalize-first"/> should be revised as <label variable="number" form="short" text-case="capitalize-first" strip-periods="true"/>

rsnbhr commented:
First of all, I sincerely apologize for the misunderstanding and any frustration caused. It was absolutely not my intention to disregard your hard work or years of expertise.

My previous commit was simply an attempt to address a few minor revisions I thought were necessary for our institute's specific requirements. However, I realize now that overriding your deliberate changes without discussion was the wrong approach.

I have now reverted the file entirely back to your version. Instead of changing the code directly, I have added comments explaining the specific adjustments I was hoping to make. If you agree with these suggestions, I would be happy to implement them myself to save you time.

Also, I have a quick question for the future: If further updates are required later on (e.g., if the institute guidelines change), what is the best process to request or submit those changes?

Best regards

POBrien333 commented:
You can absolutely make these changes that you mentioned in the various comments. Please go ahead and do so.
But your commit reverted also a variety of things I had fixed in the style. Maybe you used the old version, pre-my-changes, and hence overwrote my changes. Please go ahead and make the changes you'd want, but without touching things not related to your changes.

If in the future you want to make changes, fix errors etc., all you do is open a new PR with the fixes, only changing what you want changed and updating the date.
